string strPath
bool bView
int16 nTime
